airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -604,7 +605,12 @@ def get_cursor_field(self, column_names: Dict[str, Tuple[str, str]], in_jinja: b
                 # using an airbyte generated column
                 cursor = self.cursor_field[0]
         else:
-            raise ValueError(f"Unsupported nested cursor field {'.'.join(self.cursor_field)} for stream {self.stream_name}")
+            # Dealing with nested fields
+            cursor = reduce(
+                lambda expression, next_item: f"{expression}->'{next_item}'",
```

Comment:
not sure of this syntax with `->`... is this specific to a certain destination? which one?

Reply:
Yes, you're right, this is pg specific, it is the sintax to select the nested field in a json field. but anyway, your comments about the catalog coming from the source where enlightening, I am trying another approach.